packages/vapor/scss/guide.scss

```diff
@@ -129,6 +129,7 @@
     @import 'components/split-multiline-input';
     @import 'components/separator';
     @import 'components/table-hoc';
+    @import 'components/modal-wizard.scss';
```

Comment:
do you need the extension?

Reply:
I think you do to use the classes in the component :thinking:

Reply:
Good question! I'll see if I can remove it..

Reply:
sweet, extension was not necessary confirmed 👍 !